### PR TITLE
Improve changelog and upgrade guide

### PR DESCRIPTION
**Pull request by umulmrum**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | -
| License       | MIT

### CODE REVIEW

File: CHANGELOG-6.3.md [hunk 1]
```diff
@@ -1,8 +1,96 @@
...
+
+## PHP 7.3 Compatible
+
+The system was successfully tested for compatibility with PHP 7.3.
+
+## Cronjob Activation Toggle
+
+Cronjobs can now be disabled globally by toggling the new option `Cronjobs enabled` in the CMS settings.
+
+There are also new console commands `chameleon_system:cronjobs:enable` and `chameleon_system:cronjobs:enable` so that
```
Comment (UlrichKu) [inline CHANGELOG-6.3.md]: disable

File: UPGRADE-6.3.md [hunk 2]
```diff
@@ -3,22 +3,20 @@ UPGRADE FROM 6.2 TO 6.3
...
 
-In dev mode the debug toolbar will report deprecations concerning services that are retrieved through the
-ServiceLocator or the Symfony container directly. To be prepared for Symfony 4, ServiceLocator and container calls
-should be used as rarely as possible (no news here) and dependency injection should be preferred. Where dependency
-injection is not possible, services should be declared public explicitly. The deprecation warnings will also be
-logged, potentially leading to huge log files and if there is a large number of warnings, performance in the dev
-environment will degrade. Therefore it is recommended to deal with most of the deprecations.
+In dev mode the debug toolbar will report some deprecations concerning non-public services that are retrieved through
+the ServiceLocator or the Symfony service container directly. In Symfony 4 this will no longer be possible; in addition,
+services will be private by default in Symfony 4.
```
Comment (UlrichKu) [inline UPGRADE-6.3.md]: doubled Symfony 4

File: UPGRADE-6.3.md [hunk 3]
```diff
@@ -214,20 +259,31 @@ Log-related menu items in the backend ("logs", "log channel definition") are now
 new sidebar menu, create menu items assigned to the corresponding tables. To display these items in the classic main
 menu (which itself is no longer visible by default), assign the tables to the "Logs" content box.
 
+## Sidebar Menu Items
+
+Sidebar menu items are (in contrast to the classic main menu) created independently from tables and backend modules.
+After migration some menu items that existed in the classic main menu might not yet be migrated to new sidebar menu
+items. Create menu items manually for these tables and modules.
```
Comment (UlrichKu) [inline UPGRADE-6.3.md]: Is there a way to find out if this is the case?

File: CHANGELOG-6.3.md (hunk 1 above)
Comment (umulmrum) [reply CHANGELOG-6.3.md]: Fixed.

File: UPGRADE-6.3.md (hunk 2 above)
Comment (umulmrum) [reply UPGRADE-6.3.md]: Yes, but this redundancy makes clear that both changes are to be expected in Symfony 4. Would keep it as is.

File: UPGRADE-6.3.md (hunk 3 above)
Comment (umulmrum) [reply UPGRADE-6.3.md]: This section wasn't completely correct, so I reworded.

File: UPGRADE-6.3.md [hunk 4]
```diff
@@ -214,20 +259,32 @@ Log-related menu items in the backend ("logs", "log channel definition") are now
 new sidebar menu, create menu items assigned to the corresponding tables. To display these items in the classic main
 menu (which itself is no longer visible by default), assign the tables to the "Logs" content box.
 
+## Sidebar Menu Items
+
+Sidebar menu items are (in contrast to the classic main menu) created independently from tables and backend modules.
+After migration menu items for backend modules might not yet be migrated to new sidebar menu items. Check the backend
+module list to see if modules are missing. 
+
+All menu items from standard content boxes will be migrated automatically. Custom content boxes need to be migrated
```
Comment (UlrichKu) [inline UPGRADE-6.3.md]: There is some automatic mechanism to migrate custom content boxes?

**Pull request merged**